libevmasm/Assembly.cpp

```diff
@@ -235,10 +235,10 @@ void Assembly::assemblyStream(ostream& _out, string const& _prefix, StringMap co
 		_out << endl << _prefix << "auxdata: 0x" << toHex(m_auxiliaryData) << endl;
 }
 
-string Assembly::assemblyString(StringMap const& _sourceCodes) const
+string Assembly::assemblyString(StringMap _sourceCodes) const
 {
 	ostringstream tmp;
-	assemblyStream(tmp, "", _sourceCodes);
+	assemblyStream(tmp, "", std::move(_sourceCodes));
```

Comment:
This is not beneficial - if `_sourceCode` is not really consumed but just used, it should be taken as `const&`.